lib/ui/src/modules/ui/components/layout/index.js

```diff
@@ -1,7 +1,8 @@
-import { localStorage, window } from 'global';
+import { localStorage } from 'global';
 import PropTypes from 'prop-types';
 import React from 'react';
 import SplitPane from 'react-split-pane';
+import _ from 'lodash'; /* eslint-disable-line import/no-extraneous-dependencies */
```

Comment:
We use per-function lodash packages. You need to add `lodash.throttle` as a dependency to `package.json` in `lib/ui`, and import it like that:

```
import throttle from 'lodash.throttle';
```

By the way, that eslint warning you've suppressed was there for a reason. If you use a package that we don't have as a direct or peer dependency, it will break on user side, because they might not have this dependency

Reply:
Fixed! :)

Reply:
Is it? I still see this import and `eslint-disable`

Reply:
Oh shoot. I think I accidentally dropped that commit when I was rebasing